layouts/default.vue

```diff
@@ -463,118 +659,51 @@ function developerModeIncrement() {
       })
     }
   } else {
-    developerModeCounter++
+    developerModeCounter.value++
   }
 }
 
 async function logoutUser() {
   await logout()
 }
 
+function runAnalytics() {
+  const config = useRuntimeConfig()
+  const replacedUrl = config.public.apiBaseUrl.replace('v2/', '')
+
+  setTimeout(() => {
+    $fetch(`${replacedUrl}analytics/view`, {
+      method: 'POST',
+      body: {
+        url: window.location.href,
+      },
+    })
+      .then(() => {})
+      .catch(() => {})
+  })
+}
+function toggleMobileMenu() {
+  this.isMobileMenuOpen = !this.isMobileMenuOpen
```

Comment:
`this` refers to nothing here. There are a lot of mistakes like these in [the commit](https://github.com/modrinth/knossos/pull/1616/commits/d7a270abf690f358d991950184e6467dc8bc8a2e) to convert default layout to composition API that got chained to this PR.

```suggestion
  isMobileMenuOpen.value = !isMobileMenuOpen.value
```

Reply:
fixed